kafka-connect-s3/src/test/java/io/confluent/connect/s3/DataWriterByteArrayTest.java

```diff
@@ -228,7 +228,7 @@ protected void verifyFileListing(List<String> expectedFiles) throws IOException
 
     Collections.sort(actualFiles);
     Collections.sort(expectedFiles);
-    assertThat(actualFiles, is(expectedFiles));
+    assertEquals(actualFiles, expectedFiles);
```

Comment:
these changes (and similar ones within) will conflict with recent downstream changes. Can you please revert?